Reto 8

NJ-JEET opened the pull request:
Se incluyeron conjunto de clases que forman una calculadora básica que permite al usuario realizar operaciones matemáticas simples a través de una interfaz, con la opción de seleccionar el idioma deseado para la interacción.

NJ-JEET committed:
se agrego el diagrama de clases

NJ-JEET committed:
Se cambio la herencia de las clases en base al diagrama de clases y ademas se hicieron correciones en dichas clases. Se incluye la Potencia que anteriormente no se adjunto.
Dentro del menu se hicieron algunas modificaciones esteticas tambien.

Prof-Carlos-Castillo reviewed (changes requested):
no incluye ninguno de los requerimientos para estra practiva, favor de revisar y volver a entregar

NJ-JEET committed:
Diagrama de clases que incluyen nuevas clases pertenecientes a la traduccion del programa.

NJ-JEET committed:
Clases pertenecientes a la traduccion del menu, dentro de cada clase se encuentran diferentes funciones.
-Clase Ingles y Español: dentro de esta clses encontramos metodos abstractos que son llamados dependiendo el idioma seleccionado.
-Clase SeleccionIdioma: esta clase hace seleccion del idioma para poder mostrar el menu segun la seleccion.
-Dentro de la clase TraduccionInterfazCalculadora solo encomtramos el menu base con adaptaciones para poder trabajar con otro idioma.

NJ-JEET commented:
> no incluye ninguno de los requerimientos para estra practiva, favor de revisar y volver a entregar

Perdone profe, ya se incluyerron los cambio requeridos para la practica.
